bee-test/src/rand/receipt.rs

```diff
@@ -32,13 +29,12 @@ pub fn rand_tail_transaction_hash() -> TailTransactionHash {
 pub fn rand_migrated_funds_entry() -> MigratedFundsEntry {
     MigratedFundsEntry::new(
         rand_tail_transaction_hash(),
-        SignatureLockedSingleOutput::new(rand_address(), rand_number_range(VALID_MIGRATED_FUNDS_ENTRY_AMOUNTS))
-            .unwrap(),
+        SimpleOutput::new(rand_address(), rand_number_range(MigratedFundsEntry::AMOUNT_RANGE)).unwrap(),
     )
     .unwrap()
 }
 
-/// Generates a random ledger receipt.
-pub fn rand_ledger_receipt() -> Receipt {
-    Receipt::new(rand_receipt_payload(), rand_milestone_index())
-}
+// /// Generates a random ledger receipt.
```

Comment:
reminder

Reply:
Can't be done as part of this PR without fixing the ledger crate